workflows/prognostic_c48_run/emulation/microphysics_parameter_metadata.yaml

```diff
@@ -0,0 +1,320 @@
+air_pressure:
```

Comment:
I don't think this file is included in the `REQUIREMENTS.in` or `setup.py`. I'm kind of surprised `runfv3` works. Have you actually tried generating the data in the prognostic run image?

Reply:
The directory is copied in the dockerfile, and an then an environment variable is set in that points to it.  I haven't used the packaging route of including files before, but I think that might be a good option for a static file like this.